internal/gui/cud.go

```diff
@@ -171,7 +171,7 @@ func (g *Gui) createPersonForm() {
 			g.createPerson(form)
 		}).
 		AddButton("Cancel", func() {
-			g.closeAndSwitchPanel("form", "people")
+			g.closeAndSwitchPanel("form", "cavers") // FIXME
```

Comment:
Investigate the FIXME here

Reply:
Removing as whatever the issue was it did not present itself to me when running that code.